pa_config/rca.conf

```diff
@@ -97,5 +97,11 @@
       "field-data-cache-upper-bound" : 0.4,
       "shard-request-cache-upper-bound" : 0.05
     }
+  },
+
+  "usage-bucket-thresholds": {
+    "young-gen-heap-promotion-rate": [100.0, 250.0, 400.0],
```

Comment:
Thinking it would be more user friendly to have a key-value pair for describing these thresholds, rather than positional variables. This current design requires both users and developers to remember what each position stands for. And it is not future proof to if you add more buckets.
Something like:
```ruby
"young-gen-heap-promotion-rate": {
    "under-utilized": 20.0,
    "healthy-with-buffer": 100.0, 
    "healthy": 250.0, 
    "unhealthy": 400.0
}
```